### ISSUE TITLE
[BUG] Unifomize boilerplates about sharing original data under `/sourcedata`

### ISSUE THREAD

**Issue by oesteban** [bids-standard/bids-specification#1718]
### Describe your problem in detail.

From @Remi-Gau's proposal in [#1128 (comment)](https://github.com/bids-standard/bids-specification/pull/1128/files#r1422111146):

> How about we remove this phrase from this BEP and open a separate PR to also remove similar sentences from other data types and make sure it is expressed only once in the common principles in way we all agree on?
>
> We have done similar "refactoring" in the past when we started noticing duplication / divergence across datatypes.

This conversation was initiated from some boilerplate regarding `/sourcedata` in the context of #1128 (https://github.com/bids-standard/bids-specification/pull/1128/files#r1411290644).

### BIDS specification section

@robertoostenveld cited several instances where these statements are found along the specs (https://github.com/bids-standard/bids-specification/pull/1128/files#r1412100006).

**Comment by effigies** [comment]
I think this is already covered extensively in common principles:

https://github.com/bids-standard/bids-specification/blob/5b835b0b182b50aa27ead6dffeb727e59322d1fe/src/common-principles.md?plain=1#L241-L260

As well as:

https://github.com/bids-standard/bids-specification/blob/5b835b0b182b50aa27ead6dffeb727e59322d1fe/src/common-principles.md?plain=1#L125-L134

This macro renders a list, including:

> * `sourcedata`: A directory where to store data before harmonization, reconstruction, and/or file format conversion (for example, E-Prime event logs or DICOM files). See the [relevant section](https://bids-specification.readthedocs.io/en/stable/common-principles.html#source-vs-raw-vs-derived-data) for more information.

I vote for simple removal of this language from the modality-specific pages.

**Comment by oesteban** [comment]
Yup, that was the position I tried to explain in the context of BEP020 and I believe what @Remi-Gau proposed in more practical/precise terms at the end.

**Comment by oesteban** [comment]
I think the proposal is more like we make sure nothing relevant is lost at all if the language is removed from modality-specific files and update the common principles before removal if some detail is found to be important.
